_docs_integrate/version-5.md

```diff
@@ -25,7 +25,7 @@ required_by:
 
 The address format changed from `<3-character realm><32 or 33-character base58-string>` to `did:e:<backbone-hostname>:dids:<22-character lowercase hex string>`.
 
-## Removal of RelationshipChanges
+## Removal of RelationshipChanges {#relationshipchanges}
```

Comment:
why?

Reply:
didn't know it wasn't necessary :)

Reply:
it is necessary if you want to rewrite the anchor from `#removalofrelationshipchanges` to `#relationshipchanges` which we don't want